lld/test/MachO/link-csu-object.s

```diff
@@ -0,0 +1,14 @@
+# REQUIRES: x86
+# RUN: mkdir -p %t
+# RUN: llvm-mc -filetype=obj -triple=x86_64-apple-darwin %p/Inputs/libhello.s -o %t/hello.o
+# RUN: llvm-mc -filetype=obj -triple=x86_64-apple-darwin %s -o %t/main.o
+# RUN: %lld -L %t %t/main.o %t/hello.o -o %t/a.out
```

Comment:
Should one of these be linked with `-l` instead of by path?

Reply:
AHH

Reply:
I fixed it in my branch but the pr is already merged

Reply:
would you mind putting it in another PR? we can get it merged there. Thanks!

Reply:
> would you mind putting it in another PR? we can get it merged there. Thanks!

done https://github.com/llvm/llvm-project/pull/79018